js/header.js

```diff
@@ -23,12 +18,18 @@ headerControls_next.addEventListener('click', () => {
 })
 
 const handleNextPrevClick = (direction) => {
-    const newDate = new Date( selectedDate.getFullYear(), selectedDate.getMonth(), selectedDate.getDate() + WEEKDAYS*direction ) 
-    switchWeekView( newDate )
-    switchMonth (newDate)
-    toggleSelectedSecondary(newDate)
-    setSelectedDate( newDate )
-    updateCalendarLabels(document.querySelector('header'), selectedDate)
+    const newDate = new Date( selectedDate.value.getFullYear(), selectedDate.value.getMonth(), selectedDate.value.getDate() + WEEKDAYS*direction ) 
+    selectedDate.setState( newDate )
 }
 
-updateCalendarLabels(document.querySelector('header'), selectedDate)
+const updateCalendarLabels = ( date ) => {
+    
+    const parentElem = document.querySelector('header')
+    const monthLabel = parentElem.querySelector("[data-calendarLabel='month']")
```

Comment:
probably not much point having three selectors (one selecting parent) instead of two here